src/Internal/ConsoleApplicationResolver.php

```diff
@@ -31,6 +35,10 @@ public function findCommands(ClassReflection $classReflection): array
 
         $commands = [];
 
+        if ($this->useContainerCommandLoader) {
+            $consoleApplication->setContainerCommandLoader();
+        }
```

Comment:
Why do we really need another config option? I think this can just be appended on the `getApplication()` method below:
```php
        if ($this->application === null) {
            $this->application = (new Application(app(Container::class), app(Dispatcher::class), app()->version()))
                ->setContainerCommandLoader();
        }
```

This is essentially what the framework does, and I see no reason why anyone would want to disable this:

https://github.com/laravel/framework/blob/bc5bd660b0a3079e3fb7f392bfac06e41d80b64b/src/Illuminate/Foundation/Console/Kernel.php#L531-L541

Reply:
Well probably it makes sense, wanted to keep it without it to make sure we don't break anything but I will change it.